### ISSUE TITLE
gammu-smsd: SQL service was not compiled in!

### ISSUE THREAD

**Issue by Bowser** [gammu/gammu#141]
I am going crazy with gammu-smsd today. I tried everything, compiling and re-installing it several times.

I have already installed libmysqlclient-dev from the beginning, I tried even to upgrade it (with all other mysql related) to the unstable (sid) version of gammu.

It returns me that I didn't configure the SQL service, I can't do anything against it.
I don't understand why.

These are related params in my gammu-smsdrc configuration file.
[smsd]
Service = sql
Driver = native_mysql

This is what configure returns when I compile gammu:

root@system:/tmp/gammu# ./configure --prefix=/usr
Wrapper script for configuring CMake for Gammu.

This provides limited compatibility with configure, if you want full
configuration control, use directly CMake. More information about CMake
is available at <http://www.cmake.org>.

-- Configuring Gammu 1.36.6
-- Could NOT find Doxygen (missing:  DOXYGEN_EXECUTABLE)
-- Using mysql-config: /usr/bin/mysql_config
-- Found MySQL: /usr/include/mysql, /usr/lib/x86_64-linux-gnu/libmysqlclient.so;                                                                                       /usr/lib/x86_64-linux-gnu/libpthread.so;/usr/lib/x86_64-linux-gnu/libz.so;/usr/l                                                                                       ib/x86_64-linux-gnu/libm.so;/usr/lib/x86_64-linux-gnu/libdl.so
-- Could NOT find ODBC (missing:  ODBC_LIBRARY ODBC_INCLUDE_DIR)
-- Could not find PostgreSQL
-- libdbi not found.
-- Found CURL: /usr/lib/x86_64-linux-gnu/libcurl.so (found version "7.44.0")
-- checking for module 'glib-2.0>=2.16'
--   package 'glib-2.0>=2.16' not found
-- checking for module 'gobject-2.0'
--   package 'gobject-2.0' not found
-- checking for module 'gudev-1.0'
--   package 'gudev-1.0' not found
-- checking for module 'bluez'
--   package 'bluez' not found
-- Bluez not found.
Bluetooth support disabled, please install libbluetooth-dev or equivalent to ena                                                                                       ble Bluetooth.
-- checking for module 'libusb-1.0'
--   package 'libusb-1.0' not found
-- libusb-1.0 not found.
-- Using Linux native IrDA
-- IrDA support enabled
-- Nokia support enabled
-- Nokia MBUS2 protocol enabled
-- Nokia FBUS2 protocol enabled
-- Nokia FBUS2DLR3 protocol enabled
-- Nokia DKU2PHONET protocol enabled
-- Nokia DKU2AT protocol enabled
-- Nokia DKU5FBUS2 protocol enabled
-- Nokia FBUS2PL2303 protocol enabled
-- Nokia IRDAPHONET protocol enabled
-- Nokia FBUS2IRDA protocol enabled
-- Nokia 3320 and compatible phones support enabled
-- Nokia 650 and compatible phones support enabled
-- Nokia 61xx and compatible phones support enabled
-- Nokia 6510 and compatible phones support enabled
-- Force using 6210 frames for calendar for DCT4 phones disabled
-- Nokia 7110 and compatible phones support enabled
-- Nokia 9210 and compatible phones support enabled
-- Nokia 3650 and compatible phones support enabled
-- AT support enabled
-- AT protocol enabled
-- AT protocol over IrDA enabled
-- AT phones support enabled
-- Alcatel support enabled
-- Alcatel protocol enabled
-- Alcatel phones support enabled
-- OBEX compatible phones support enabled
-- OBEX protocol over IrDA enabled
-- Generic OBEX phones support enabled
-- AT with OBEX phones support enabled
-- Symbian phones support (S60) enabled
-- Symbian phones support (gnapplet) enabled
-- GNAPBUS protocol over IrDA enabled
-- Gnapplet phones support enabled
-- S60 phones support enabled
-- Cell Broadcast messages support enabled
-- Backup/Restore functions enabled
-- Init script not being installed, install it by -DINSTALL_LSB_INIT=ON
-- Udev rules not being installed, install it by -DINSTALL_UDEV_RULES=ON
-- Configuring done
-- Generating done
-- Build files have been written to: /tmp/gammu/build-configure

Please help me, I need to work on it as soon as possible

**Comment by nijel** [comment]
What's output of ``gammu-smsd -v``?

**Comment by Bowser** [comment]
So, at the end the problem was resolved with a complete uninstall, looking in the FS for each file related to gammu (maybe an uninstall option for make could help in the future). I had to resolve it as soon as possible, so now I'm unable to investigate on what happened.
I don't know why that day it decided not to work anymore, giving problems even with compiling and installing again some times from source.
Thank you anyway.
